### PR TITLE
added CI appveyor.com

### PR DESCRIPTION
**Pull request by merlokk**
Lets put file to root and then all contributors can config appveyor.com to test releases.
Admin can config CI for main repository, but it not necessary)

### CODE REVIEW

**Comment by merlokk** [comment]
How it works:
1. Get ProxSpace from Gator's repository
2. Get proxmark commit/PR/etc...
3. Compile proxmark
4. Make release.zip with binary proxmark files
5. Tests proxmark (help and hardnested)

Maybe it needs to think about adding more logic tests....  and rin them with proxmark3 -test (as sample...)

**Comment by iceman1001** [comment]
Nice,  so you got a working appveyor file...   I'm jealous now.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    } else {
+      Write-Host "Tests [ ERROR ]" -ForegroundColor Red
+      throw "Tests error."
+    }
+on_success:
+- ps: Write-Host "Build success..." -ForegroundColor Green
+on_failure:
+- ps: Write-Host "Build error." -ForegroundColor Red
+on_finish:
+- ps: $blockRdp = $false; iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
```
Comment (pwpiwi) [inline appveyor.yml]: Does this work without the NewLine?

**Comment by pwpiwi** [comment]
Arrrrgh. Please revert.

File: appveyor.yml (hunk 1 above)
Comment (merlokk) [reply appveyor.yml]: this file works. Its export from their config tool
let me add a line to the end

**Comment by merlokk** [comment]
reset my changes to adding file?

**Comment by pwpiwi** [comment]
Revert my push (merge) to your repository.

**Comment by merlokk** [comment]
done

**Pull request merged**